lib/rules/acronyms.js

```diff
@@ -20,7 +20,7 @@ const bothOrNeither = (a, b) => a === b;
  */
 exports.check = (word, { caseInsensitive }) => caseInsensitive ? false : ACRONYM.test(word.split(' ')[0]);
 
-exports.run = (word, opts) => {
+exports.run = (word, _opts) => {
```

Comment:
Looks like _opts isn't used here?

Reply:
I'm gonna guess this is because this is the signature for a rule's `run` method, so you went for consistency, but I don't actually have a problem dropping it here if it's not used.

Reply:
ok - have pushed that change.